### PR TITLE
Add release calendar

### PR DESCRIPTION
**Pull request by spastorino**
r? @davidtwco

I'm not sure if we want this as a infra event, unsure about the description, etc.

### CODE REVIEW

**Comment by spastorino** [comment]
cc @Mark-Simulacrum

**Comment by Mark-Simulacrum** [comment]
I have a personal recurring event for this, but not opposed to adding a calendar for it. Maybe it should be it's own calendar though that has nothing else on it? Not really sure who the audience for it is though, especially given how easy it is to create this (just 6 week recurrence)...

**Comment by spastorino** [comment]
In the current PR is a separated calendar, that is "owned" by t-infra. Yeah, I understand what you say but still I think it's useful. A side possibility this gives is the way to create new events for point releases.

File: release.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name = "Rust Releases Calendar"
+description = "New releases of Rust"
+
+[[events]]
+uid = "1706194117478"
+title = "New Rust Release"
+description = ""
+location = ""
```
Comment (davidtwco) [inline release.toml:8]:
```suggestion
```

I think these are optional

**Review by davidtwco — APPROVED** [review]

**Pull request merged**